knees4bees/dream-up-pup
# Feature/write story page

### What does this PR do?
Creates landing page and create-story page (satisfies user story #1)

### Are there any known bugs?
No bugs in functionality but the styling needs a lot of work.

### Summary of the changes
- Creates `Create` component with title input, three panels for story images/text, and "Create Story" button
- Fetches three random images of breed selected from drowdown list on landing page

### Notes for the reviewer
As mentioned above, the styling needs a LOT of work at this point. But the functionality on the landing page and on the create-story page should all be there, with the user inputs being held in state (controlled inputs).

### How should this be tested?
Confirm that the landing page loads without errors, that you are able to select a breed from the dropdown with that selected breed being held in state in the `App` component (use dev tools to check this), that when you click the "Write Story" button you are taken to a new page `/create` and three images of your selected breed appear with text input boxes below them, in which you can enter your own text that is held in state in the `Create` component (use dev tools to check).
